src/vmm/src/lib.rs

```diff
@@ -1005,6 +1006,44 @@ impl Vmm {
         Ok(entry_addr)
     }
 
+    /// Loads the initrd from a file into the given memory slice.
```

Comment:
>  Is there a technical reason for which this is needed in vmm/src/lib.rs

This is because the load address of the initrd is computed in the arch module, and I wanted to avoid a dependency of the `kernel` module from `arch`.